chainer_chemistry/models/gwm.py

```diff
@@ -5,11 +5,145 @@
 from chainer_chemistry.links import GraphLinear
 
 
+class WarpGateUnit(chainer.Chain):
+    def __init__(self, output_type='graph', hidden_dim=16,
+                 dropout_ratio=-1, activation=functions.sigmoid):
+        super(WarpGateUnit, self).__init__()
+        if output_type == 'graph':
+            LinearFunc = GraphLinear
+        elif output_type == 'super':
+            LinearFunc = links.Linear
+        else:
+            raise ValueError
```

Comment:
show proper error message.
`ValueError('output_type = {} is unexpected. graph or super is supported.'.format(output_type))`

Reply:
done in new PR